routes/articles.js

```diff
@@ -0,0 +1,100 @@
+const express = require('express');
+const router = express.Router();
+const articleDB = require('../db/articles.js');
+const middleware = require('../middleware/validate');
+const database = articleDB.getArticles().articles;
+
+/************************
+ *  GET
+************************/
+
+router.get('/', (req, res) => {
+  const articles = articleDB.getArticles();
+  if (articles.length < 1) {
+    articles.message = "No products in our database";
+  } else {
+    articles.message = "";
+  }
+
+  res.status(200);
+  return res.render('./articles/index', articles);
+});
+
+router.get('/new', (req, res) => {
+  res.status(200);
+  return res.render('./articles/new', { message: "" });
+});
+
+router.get('/:title', middleware.validator, (req, res) => {
+  const params = req.params;
+  const articleIndex = articleDB.findArticle(params.title);
+
+  const data = {
+    title: database[articleIndex].title,
+    author: database[articleIndex].author,
+    body: database[articleIndex].body
+  }
+
+  res.status(200);
+  return res.render('./articles/article', data);
+});
+
+router.get('/:title/edit', middleware.validator, (req, res) => {
+  let params = req.params;
+  let articleIndex = articleDB.findArticle(params.title);
```

Comment:
This should use a function to get a single article, please create one if you don't have one.